### DISCUSSION TITLE
Which dotnet version should be used?

### DISCUSSION THREAD

**Discussion by davemap** [skalahonza/TidepoolToNightScoutSync#10] [Q&A] [answered]
Hi, 

Thanks for creating this tool. I am trying to use a mac to upload from tidepool to nightscout but I'm getting null warnings when running the tool. It then breaks and I can't run it again. I have dotnet v7, which version are you running? I may also try it on a windows machine too. 

Thanks,
David

**Comment by davemap** [comment]
<img width="1705" alt="image" src="https://user-images.githubusercontent.com/47156308/210456344-d09d255e-4ccf-4996-b3bc-cf60d12c97c9.png">

**Reply by skalahonza** [reply]
Hi. The tool uses .net core 3.1. The warnings however should not be the cause of the crash.

**Comment by skalahonza** [comment, marked as answer]
As of today, use [.NET SDK 8](https://dotnet.microsoft.com/en-us/download)
